phpBB/phpbb/filesystem/exception/filesystem_exception.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\filesystem\exception;
+
+class filesystem_exception extends \phpbb\exception\runtime_exception implements \phpbb\exception\exception_interface
```

Comment:
the interface is useless here (already implemented by \phpbb\exception\runtime_exception)